src/main/resources/templates.xsd

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    This schema describes the dialects XML file for a benchmark
+
+    If you modify this schema, you must regenerate the jaxb
+    bindings. It's easy to do:
+
+    cd src/
+    xjc -p com.oltpbenchmark.api.templates com/oltpbenchmark/api/templates.xsd
```

Comment:
Can we automate this too?  e.g. run it during the CI pipelines and make sure that no file changes are generated (e.g. with a `git diff --porcelain` check and abort the checkin since local changes are out of sync.

Reply:
Other option would be to always autogenerate those as a part of the build and avoid checking them into the repo at all.

Reply:
I think I copied this over from com/oltpbenchmark/api/dialect.xsd which is what you wanted me to mimic. I don't think that's part of the CI pipeline right now and I'm not sure we could auto-generate it as part of the pipeline.

Reply:
Fair.  We can probably leave that for a separate improvement.

Reply:
Reopening so I remember to make a new Issue for this.